### PR TITLE
Make updates to get CI build fully passing

### PR DESCRIPTION
**Pull request by phil-allen-msft**
After merging previous changes, found that local build tends towards EFTools.sln, but the CI is using EFTools.msbuild.  Replicated build command for EFTools.msbuild and several listed after it in the yml for CI.

### CODE REVIEW

File: EF.msbuild [hunk 1]
```diff
@@ -11,12 +11,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectToBuild Include="src\EntityFramework\EntityFramework.csproj" />
```
Comment (maryamariyan) [inline EF.msbuild:14]: just to clarify, reason for these projects being removed, were these projects deprecated or there are new project or components to replace them?

**Comment by maryamariyan** [comment]
> After merging previous changes, found that local build tends towards EFTools.sln, but the CI is using EFTools.msbuild. Replicated build command for EFTools.msbuild and several listed after it in the yml for CI.

How do these changes affect the CI pipeline? Have some CI configurations been updated too?

File: src/EFTools/setup/EFToolsMsi/Defines.wxi [hunk 2]
```diff
@@ -1,27 +0,0 @@
...
-    <?define UpgradeCode = {27790FD4-0395-45E1-9309-447B91604CCA}?>
-    <?define FullProductName = !(loc.FullProductName_Dev11)?>
-    <?define ProductDescription = !(loc.ProductDescription_Dev11)?>
-    <?define ProductVersion = 11.2.$(var.BuildRevisionProductVersion)?>
-  <?elseif $(var.VisualStudioVersion) = 12.0?>
-    <?define UpgradeCode = {B9BF703A-2405-4A71-AFC5-38F59C11911D}?>
-    <?define FullProductName = !(loc.FullProductName_Dev12)?>
-    <?define ProductDescription = !(loc.ProductDescription_Dev12)?>
-    <?define ProductVersion = $(var.AssemblyFileVersion)?>
-  <?elseif $(var.VisualStudioVersion) = 14.0?>
```
Comment (maryamariyan) [inline src/EFTools/setup/EFToolsMsi/Defines.wxi:16]: last question, are there any concerns with removing support for older Visual Studio versions?

**Review by maryamariyan — APPROVED** [review]

File: EF.msbuild (hunk 1 above)
Comment (phil-allen-msft) [reply EF.msbuild:14]: The real code is in https://github.com/dotnet/ef6 (and it has been getting updates) , this was a copy to make certain methods available for writing some specific unittests.  We're moving forwards the version of the assemblies, so there are going to be differences and after weighing the alternatives it became "use the modern versions from the nuget packages"

**Comment by phil-allen-msft** [comment]
> > After merging previous changes, found that local build tends towards EFTools.sln, but the CI is using EFTools.msbuild. Replicated build command for EFTools.msbuild and several listed after it in the yml for CI.
> 
> How do these changes affect the CI pipeline? Have some CI configurations been updated too?

No change in CI configuration; it was a matter of local builds were done with the SLN file open, but the CI uses the EFTools.msbuild which calls  EFTools.sln.  Success in the SLN did not translate into success in the CI, and at the time the "check" for a passing CI build was disabled (now restored).

File: src/EFTools/setup/EFToolsMsi/Defines.wxi (hunk 2 above)
Comment (phil-allen-msft) [reply src/EFTools/setup/EFToolsMsi/Defines.wxi:16]: It's a good question.  At this point, we have accepted package versions that are higher than 14 (and indeed, are really in 17.7 or higher, given the contents of https://github.com/dotnet/ef6tools/blob/36aa191f56cbf07ca555ee1fdb07ed7eed1376be/Directory.Build.props).  If we need to go back and service earlier versions, we'll need to start from a previous commit.

**Pull request merged**